### PR TITLE
Markodonnell deriv patch 2

### PR DESCRIPTION
**Pull request by markodonnell-deriv**

### CODE REVIEW

File: cypress/.DS_Store
Comment (habib-deriv) [inline cypress/.DS_Store:1]: Do we need this? Or should add this in .gitignore?

File: cypress/results/mochawesome.json [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+              "fullTitle": "Home page link tests Markets - menu",
+              "timedOut": null,
+              "duration": 6961,
+              "state": "passed",
+              "speed": "medium",
+              "pass": true,
+              "fail": false,
+              "pending": false,
+              "context": null,
+              "code": "//tag: smoke\nif (Cypress.env('isMobile') == 'true') {\n  //Click through 'Markets' menu-items. \n  cy.findByRole('img', {\n    name: 'hamburger menu'\n  }).click();\n  cy.findByRole('button', {\n    name: 'Markets chevron'\n  }).click();\n  cy.findByRole('link', {\n    name: 'Forex Forex Trade the world’s largest financial market with popular forex pairs.'\n  }).click();\n  cy.findByRole('heading', {\n    name: 'Forex',\n    exact: true\n  }).should('be.visible');\n} else {\n  cy.findByRole('link', {\n    name: 'Who we are'\n  }).click();\n  cy.findByRole('heading', {\n    name: 'Who we are'\n  }).should('be.visible');\n}",
```
Comment (habib-deriv) [inline cypress/results/mochawesome.json:48]: "Forex Forex" is used twice

File: cypress/e2e/markets.cy.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    })
+
+    it('Markets - menu', () => {
+      //tag: smoke
+      if (Cypress.env('isMobile') == 'true')
+      {
+        //Click through 'Markets' menu-items. 
+        cy.findByRole('img', { name: 'hamburger menu' }).click()
+        cy.findByRole('button', { name: 'Markets chevron' }).click()
+        cy.findByRole('link', { name: 'Forex Forex Trade the world’s largest financial market with popular forex pairs.' }).click()
```
Comment (habib-deriv) [inline cypress/e2e/markets.cy.js:16]:
```suggestion
        cy.findByRole('link', { name: 'Forex Trade the world’s largest financial market with popular forex pairs.' }).click()
```

File: cypress/.DS_Store
Comment (markodonnell-deriv) [reply cypress/.DS_Store:1]: yes, we should

File: cypress/e2e/markets.cy.js (hunk 2 above)
Comment (markodonnell-deriv) [reply cypress/e2e/markets.cy.js:16]: The name of the link is strange but that's what it is unfortunatley. If we change the locator it won't be found. of course, these kinds of things can go back to the dev team for root cause change, eventually.

**Review by habib-deriv — APPROVED** [review]

**Pull request merged**